README.md

```diff
@@ -73,32 +73,53 @@ However, care should be taken here because if the schema has been migrated in th
 See [here](docs/DisasterRecovery.md)
 
 ## Passwords
+
+Database passwords are managed by the vault and laid out ascii
+
+```
+/secret/database/:password_group/users/:username
+```
+
+Currently our two password groups are `science` and `production`
+
 ### Retrieve
-To get a database password for use with postgres, there is a helper script that
-can be used like so:
+
+To get a database password for use with postgres
 
 ```
-export PGPASSWORD=$(./scripts/get_db_password.sh readonly)
+export PGPASSWORD=$(vault read -field=password secret/database/science/users/readonly)
 psql -h support.montagu.dide.ic.ac.uk -U readonly -d montagu
 ```
 
-This gets the password for the `readonly` user. To see all users, use
-`vault list secret/database/users`
+This gets the password for the `readonly` user for the `science` password group (which is suitable for the database running on support.montagu.dide.ic.ac.uk). To see all users, use
 
-### Generate
+```
+vault list secret/database/science/users
+vault list secret/database/production/users
+```
+
+### Add a new user
 
-Passwords can be generated by running the script
+Run
 
 ```
-./scripts/generate_passwords.sh
+./src/generate_passwords.py add_user <username>
 ```
 
-At present this manages only the database passwords, with the layout of
+Passwords can be regenerate by running the script
 
 ```
-/secret/database/:machine/users/:username
+./src/generate_passwords.py <group>
```

Comment:
Missing an argument to the script here, I think

Reply:
thanks